src/components/common/SearchBar.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+
+class SearchBar extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { term: '' };
+    this.onInputChange = this.onInputChange.bind(this);
+    this.onFormSubmit = this.onFormSubmit.bind(this);
+  }
+
+  onInputChange(event) {
+    this.setState({ term: event.target.value });
+  }
+  onFormSubmit(event) {
+    event.preventDefault();
+    //Need to call an action here to update application state 'searchQuery'
+    console.log('You typed ', this.state.term);
+    this.setState({ term: '' });
+  }
+  render() {
+    return (
+      <form onSubmit={this.onFormSubmit} className="input-group">
+        <input
+          placeholder="Find healthy food"
+          className="form-control"
+          value={this.state.term}
```

Comment:
The search term needs to be stored in the Redux store... that is one of the main differences with React + Redux vs React, you rarely use states for components and actions dispatched and reducers written to handle updating the state. Please refer to the redux tutorial link if you are confused.